torch/csrc/inductor/aoti_runtime/model_container.h

```diff
@@ -467,14 +467,34 @@ class AOTInductorModelContainer {
           constants_blob_ptr + constants_internal_offset_[idx];
       void* user_constant_ptr;
       int64_t constant_size;
+      int64_t* stride;
+      int64_t offset;
       aoti_torch_get_data_ptr(tensor, &user_constant_ptr);
       aoti_torch_get_storage_size(tensor, &constant_size);
+      AOTI_TORCH_ERROR_CODE_CHECK(aoti_torch_get_strides(tensor, &stride));
+      AOTI_TORCH_ERROR_CODE_CHECK(
+          aoti_torch_get_storage_offset(tensor, &offset));
+      auto dtype = models_[0]->constant_dtype(idx);
+
 #ifdef USE_XPU
       sycl::queue* queue_ptr = nullptr;
       aoti_torch_get_current_sycl_queue((void**)&queue_ptr);
       queue_ptr
           ->memcpy(internal_constants_ptr, user_constant_ptr, constant_size)
           .wait();
+#elif USE_MPS
+      internal_constants_ptr = constants_blob_ptr;
+      aoti_torch_mps_copy_buffer(
+          constants_blob_ptr,
+          constants_internal_offset_[idx],
+          offset,
+          constant_size,
+          user_constant_ptr);
```

Comment:
Hmm, I still don't think this is correct based on this signature, `aoti_torch_mps_copy_buffer(void* buffer, size_t constant_offset, size_t bytes_read, size_t data_size, void* constant_buffer)`. What is the difference between `bytes_read` and `data_size`, and why using `offset` as `bytes_read`?